cfme/fixtures/pytest_selenium.py

```diff
@@ -889,7 +889,7 @@ def checkbox(loc, set_to=False):
             selected = el.is_selected()
 
         if selected is not set_to:
-            logger.debug("Setting checkbox %s to %s", str(loc), str(set_to))
+            logger.debug("Setting checkbox to %s", str(set_to))
```

Comment:
Consider keeping the same logging and switching to use format() instead of attempting to cast vars to string. This should handle loc being a WebElement.